package.json

```diff
@@ -4,12 +4,12 @@
   "description": "Compositional JSON encode/decode library for BuckleScript",
   "main": "index.js",
   "scripts": {
-    "build": "bsb -make-world",
-    "start": "bsb -make-world -w",
+    "build": "./node_modules/.bin/bsb -make-world",
```

Comment:
did that so it'll build with the local bucklescript for this package, and not be overridden by global installation

Reply:
I actually think I prefer to have it as it is, as I then only have to upgrade the global install and will get a warning if there's a version mismatch between local and global. Using this, if the local install is out of date, the result will just be wrong.